acme/setup.py

```diff
@@ -12,6 +12,7 @@
     'cryptography>=0.8',
     # Connection.set_tlsext_host_name (>=0.13), X509Req.get_extensions (>=0.15)
     'PyOpenSSL>=0.15',
+    'dnspython',
```

Comment:
Unfortunately, this adds additional dependency to the project. Moreover, some of the clients using this library (e.g. https://github.com/kuba/simp_le) will never use DNS challenge. Unless there are some technical arguments against, I would strongly prefer to move `dnspython` to `extras_require` (say `dns`) and make sure the rest of the code is importable when `dnspython` is not installed (i.e. catching `ImportError` or alternatively importing at the non-module scope - I'm not sure what's better here).